src/DssSpell.sol

```diff
@@ -50,8 +72,186 @@ contract DssSpellAction is DssAction {
+    address internal constant TRUENAME_WALLET            = 0x612F7924c367575a0Edf21333D96b15F1B345A5d;
+    address internal constant UPMAKER_WALLET             = 0xbB819DF169670DC71A16F58F55956FE642cc6BcD;
+    address internal constant WBC_WALLET                 = 0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47;
+    address internal constant LIBERTAS_WALLET            = 0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f;
+    address internal constant CODEKNIGHT_WALLET          = 0xf6006d4cF95d6CB2CD1E24AC215D5BF3bca81e7D;
+    address internal constant PBG_WALLET                 = 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2;
+    address internal constant FLIPFLOPFLAP_WALLET        = 0x3d9751EFd857662f2B007A881e05CfD1D7833484;
+    address internal constant BANDHAR_WALLET             = 0xE83B6a503A94a5b764CCF00667689B3a522ABc21;
+
+    // -- DAI TRANSFERS --
+    address internal constant BLOCKTOWER_WALLET          = 0xc4dB894A11B1eACE4CDb794d0753A3cB7A633767;
+
+    // Function from https://github.com/makerdao/spells-goerli/blob/7d783931a6799fe8278e416b5ac60d4bb9c20047/archive/2022-11-14-DssSpell/Goerli-DssSpell.sol#L59
+    function _updateDoc(bytes32 ilk, string memory doc) internal {
+        ( , address pip, uint48 tau, ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+        require(pip != address(0), "DssSpell/unexisting-rwa-ilk");
+
+        // Init the RwaLiquidationOracle to reset the doc
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            ilk, // ilk to update
+            0,   // price ignored if init() has already been called
+            doc, // new legal document
+            tau  // old tau value
+        );
+    }
+
     function actions() public override {
+        // --- Activate Andromeda Autoline ---
+        // Forum: https://forum.makerdao.com/t/consolidated-action-items-for-2023-06-28-executive/21187
+        // Forum: https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974
+
+        // Activate autoline with line 1.28 billion DAI, gap 50 million DAI, ttl 86400
+        DssExecLib.setIlkAutoLineParameters("RWA015-A", 1_280 * MILLION, 50 * MILLION, 24 hours);
+
+        // Bump Oracle Price to 1.28 billion DAI
+        // Debt ceiling * [ (1 + RWA stability fee ) ^ (minimum deal duration in years) ] * liquidation ratio
+        // As we have SF 0 for this deal, this should be equal to ilk DC
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).bump(
+                "RWA015-A",
+                 1_280 * MILLION * WAD
+        );
+        DssExecLib.updateCollateralPrice("RWA015-A");
+
+
+        // --- Initialize New Andromeda OutputConduit ---
+        // Poll: https://forum.makerdao.com/t/consolidated-action-items-for-2023-06-28-executive/21187
+
+        // OPERATOR permission on RWA015_A_OUTPUT_CONDUIT
+        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).hope(RWA015_A_OPERATOR);
+        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).mate(RWA015_A_OPERATOR);
```

Comment:
As with test related comment in the review above.

I would follow "better safe than sorry" approach and disable the rights for the old conduit - because it does not make a real difference in case it all works as expected and because it's just 3 lines of code. And in case it does not work as expected for whatever reason - we have an extra thing that ensures access rights.

Reply:
I don't mind doing that, but this statement is not correct:
 - This is not an MCD core contract
 - Conduit doesn't have any permissions on MCD core contracts
 - This contract is more like a  helper for the party minting DAI from the URN (they don't need to manually swap DAI to USDC/USDP/any PSM)
Let me know if you still think this contract can lead to some attack vector, if yes I am happy to unwind OPERATOR permissions

Reply:
Makes sense, i still do **not** see any attack vector, but it does not mean there's none :) .

Let's please unwind the permissions even if it is absolutely not needed.

Reply:
Sure,  no problem

Reply:
Done

Reply:
Issue looks resolved but just wanted to add that an RWA conduit _is_ a core contract where it is deployed by Maker and/or is in the Chainlog.